src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1753,4 +1753,7 @@ This version used in: {2}</value>
   <data name="Value_colon" xml:space="preserve">
     <value>Value:</value>
   </data>
+  <data name="Implement_through_0" xml:space="preserve">
+    <value>Implement through {0}</value>
```

Comment:
```suggestion
    <value>Implement through '{0}'</value>
```

I really there being a clear indication when referring to code in fix messages. We also do this with keywords in messages.

Reply:
done!